GitCommands/Settings/AppSettings.cs

```diff
@@ -336,6 +336,7 @@ public static string GitCommandValue
             }
             set
             {
+                GitVersion.ResetVersion();
```

Comment:
Only if value has changed?

Reply:
I considered that, but wanted to keep it simple.
It is not the end of the world to run the command again....
Can change